### PR TITLE
[DOCS] Add usage chapter

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

**Comment by andreaskienast** [comment]
Thanks!

File: Documentation/Index.rst [hunk 1]
```diff
@@ -67,6 +67,32 @@ For compatibility reasons, this extension is installable in TYPO3 v11 already.
...
+
+   <?php
+   return [
+       // Icon identifier
+       'myfontawesomeicon' => [
+           'provider' => \FriendsOfTYPO3\FontawesomeProvider\Imaging\IconProvider\FontawesomeIconProvider::class,
+           // The FontAwesome icon name
+           'name' => 'spinner',
+           // Additional css classes
+           'additionalClasses' => 'fa-fw',
```
Comment (andreaskienast) [inline Documentation/Index.rst]: Do you have a hint on where this is used? I cannot find this in the icon provider, nor in Core. I see this is documented at https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/ApiOverview/Icon/Index.html which looks wrong to me.

Comment (brotkrueml) [reply Documentation/Index.rst]: I just migrated the usage from the mentioned documentation. But you are right, "additionalClasses" is not available in the code. I will provide an update of the patch and take care of the "TYPO3 Explained" documentation.

**Review by andreaskienast — APPROVED** [review]
Thanks again for providing more documentation.

**Pull request merged**